Tests/FoundationEssentialsTests/GregorianCalendarRecurrenceRuleTests.swift

```diff
@@ -426,6 +426,29 @@ final class GregorianCalendarRecurrenceRuleTests: XCTestCase {
         ]
         XCTAssertEqual(results, expectedResults)
     }
+
+    func testYearlyRecurrenceMovingToFeb29() {
+        /// Rule for an event that repeats on February 29th of each year, or closest date after
+        var rule = Calendar.RecurrenceRule(calendar: gregorian, frequency: .yearly, matchingPolicy: .nextTimePreservingSmallerComponents)
+        rule.months = [2]
+        rule.daysOfTheMonth = [29]
+
+        let rangeStart = Date(timeIntervalSince1970: 946684800.0) // 2000-01-01T00:00:00-0000
+        let rangeEnd = Date(timeIntervalSince1970: 1262304000.0) // 2010-01-01T00:00:00-0000
+        var birthdays = rule.recurrences(of: rangeStart, in: rangeStart..<rangeEnd).makeIterator()
+        //                               ^ The initial date here does not matter, since the rule will change the month and day
```

Comment:
I thought the initial date does matter in a sense -- it does affect hour, minute and second of the results. Perhaps rephrase this comment to clarify that the initial date does not affect specifically the month and day of the results?